index.bs

```diff
@@ -636,6 +636,12 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. [=list/Remove=] all entries in |cache| where the entry's [=attribution source/expiry time=] is less than the current time.
 1. If the [=list/size=] of |cache| is greater than or equal to the user agent's
      [=max source cache size=], return.
+1. Let |pendingSourcesForSourceOrigin| be the [=set=] of all
+    [=attribution sources=] |pendingSource| of |cache| where |pendingSource|'s
+    [=attribution source/source origin=] and |source|'s
+    [=attribution source/source origin=] are equal.
```

Comment:
Should we say "same origin" here to be precise?
https://html.spec.whatwg.org/multipage/origin.html#same-origin

We already disallow opaque origins so it should be equivalent, and I don't think there's a standard "origin equality"